Riot/Modules/Contacts/DataSources/ContactsDataSource.m

```diff
@@ -548,7 +548,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         if (indexPath.row < filteredLocalContacts.count)
         {
             contact = filteredLocalContacts[indexPath.row];
-            showMatrixIdInDisplayName = YES;
+            // Tchap: Force to hide MatrixID in DisplayName
+//            showMatrixIdInDisplayName = YES;
```

Comment:
The right code here in Element-iOS should be:
`showMatrixIdInDisplayName = self.forceMatrixIdInDisplayName`

I will discuss with you about a suggestion to improve forceMatrixIdInDisplayName property use in Element-iOS